docs/docs/animations/reanimated.md

```diff
@@ -89,4 +89,50 @@ export const AnimationWithTouchHandler = () => {
     </GestureDetector>
   );
 };
+```
+
+## Reanimated 2
+
+If you are using Reanimated 2, we offer a hook named `useSharedValueEffect` where you can update Skia values when Reanimated values are updated.
+
+:::info
+
+useSharedValueEffect() runs on the JS thread. For UI-thread animated, see [Reanimated 3 support](#reanimated).
```

Comment:
This is a bit wrong - in the example below everything would be running on the UI thread - no JS thread will be even visited. Maybe we should rephrase this and focus less on threads in the docs?

Reply:
The goal of the statement is to explain that while we support Reanimated 2, the integration is quite different from v3 and we recommend people to use v3 if possible. Definitely open to rephrase that.

Reply:
I agree about the goal, info is key here :) What about something like:

```suggestion
useSharedValueEffect() will also run on the main thread but is not as convenient as using Reanimated 3's shared values directly as property values in Skia elements, so we suggest using Reanimated 3 if possible.
```